sale_order_import/tests/test_sale_order_import.py

```diff
@@ -0,0 +1,241 @@
+        # My Company,Test Note,Free delivery charges,Sales,+1234567890
+
+        # Compares the values with test_sale_order.csv file.
+        self.assertEqual(
+            sale_order[0].partner_id,
+            self.partner_02,
+            'From test Sale Order file the "customer"'
+            " does not match with test records",
+        )
+        self.assertEqual(
+            sale_order[1].partner_id,
+            self.partner_01,
+            'From test Sale Order file the "customer"'
+            " does not match with test records",
+        )
+        self.assertEqual(
+            sale_order[0].partner_invoice_id,
+            self.partner_02,
+            "Partner Invoice field does not match with Test Partner",
+        )
+        self.assertEqual(
+            sale_order[1].partner_invoice_id,
+            self.partner_01,
+            "Partner Invoice field does not match with Test Partner",
+        )
+        self.assertEqual(
+            sale_order[0].pricelist_id,
+            self.public_pricelist,
+            "From test Sale Order file the pricelist"
+            " field data does not match with records",
+        )
+        self.assertEqual(
+            sale_order[0].partner_shipping_id,
+            self.partner_02,
+            "Partner shipping field does not match with Test Partner",
+        )
+        self.assertEqual(
+            sale_order[1].partner_shipping_id,
+            self.partner_01,
+            "Partner shipping field does not match with Test Partner",
+        )
+        self.assertEqual(
+            sale_order[0].payment_term_id,
+            self.partner_02.property_payment_term_id,
+            "Sale Order Payment term field data does not match"
+            " with Test Partner Payment term condtion",
+        )
+        self.assertEqual(
+            sale_order[0].picking_policy,
+            "direct",
+            'Sale order picking policy value does not match with "direct"',
+        )
+        self.assertEqual(
+            sale_order[0].team_id,
+            self.team,
+            'From test Sale Order file the "Team"' " does not match with test records",
+        )
+        self.assertEqual(
+            sale_order[0].carrier_id.id,
+            self.free_delivery,
+            'From test Sale Order file the "Carrier"'
+            " does not match with test records",
+        )
+        self.assertEqual(
+            sale_order[0].warehouse_id,
+            self.warehouse,
+            'From test Sale Order file the "warehouse" '
+            "does not match with test records",
+        )
+        self.assertEqual(
+            sale_order[0].note,
+            self.note,
+            "From test Sale Order file the customer does not match with records",
+        )
+
+    def test_update_error_log_02(self):
+        """
+            This test evaluates the import test sales order CSV
+             file and it update the error log records if any issue occurred while import
```

Comment:
This is not what this test is actually doing. The test is not using the sales order CSV file at all.